dataflow/data/bigquery/batch-runs/historic/uploads_ip_by_day_base.json

```diff
@@ -1,14 +1,14 @@
 {
-  "projectId": "mlab-oti",
+  "projectId": "mlab-sandbox",
   "queryFile": "./data/bigquery/queries/base_uploads_ip_by_day.sql",
   "schemaFile": "./data/bigquery/schemas/base_uploads_ip.json",
-  "outputTable": "bocoup_prod.base_uploads_ip_by_day",
+  "outputTable": "data_viz.base_uploads_ip_by_day",
 
-  "lastDateFromTable": "[mlab-oti:plx.google:m_lab.ndt.all]",
+  "lastDateFromTable": "[plx.google:m_lab.ndt.all]",
```

Comment:
To be fair, we are not using the `lastDateFromTable` feature (auto end date detection) currently since it requires command line arguments at the moment.